utils/plot-tasks.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name
+"""Script to plot tasks from profiling data.
+
+This script requires the matplotlib and numpy Python modules.
+"""
+
+from __future__ import print_function
+# mathplotlib does not support Unicode strings a column names.
```

Comment:
Isn't this just a problem when setting the column names in pyplot then? Using byte strings here is going to cause compatibility problems in the near future. Please reconsider this.

Reply:
if I recall correctly, bytes column names cause issues for python3 and unicode string for python2. For now it is not problematic if these scripts only support 1 python version.